packages/react-core/src/components/Pagination/PaginationOptionsMenu.tsx

```diff
@@ -192,15 +193,15 @@ export const PaginationOptionsMenu: React.FunctionComponent<PaginationOptionsMen
   );
 
   const menu = (
-    <Menu className={css(className)} onSelect={onSelect} ref={menuRef}>
+    <Menu className={css(className, styles.optionsMenuMenu)} onSelect={onSelect} ref={menuRef}>
```

Comment:
@srambach is there a way to do this without using the deprecated options menu?

Reply:
@tlabaj yes, we'll need to reconfigure some [pagination component styles](https://github.com/patternfly/patternfly/blob/v5/src/patternfly/components/Pagination/pagination.scss) because they're currently targeting the options menu component specifically. Is this something we want to try and get in before alpha testing?